packages/plasma

```diff
@@ -0,0 +1,10 @@
+plasma6-plasma
+sddm
+firefox
```

Comment:
This is duplicated, like @pkgdemon mentioned.

Reply:
@SunilSherkhane When I asked about moving firefox into common that was not done.